src/north/filewriter/FileWriter.class.js

```diff
@@ -21,6 +21,7 @@ class FileWriter extends ApiHandler {
     this.outputFolder = path.resolve(applicationParameters.FileWriter.outputFolder)
     this.prefixFileName = applicationParameters.FileWriter.prefixFileName ?? ''
     this.suffixFileName = applicationParameters.FileWriter.suffixFileName ?? ''
+    this.connected = true
```

Comment:
`this.connected = true` should it be instead set if the connect() has been successful in the parent class?